lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && ((this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) || routeInfo.urlInfo.query.noop === 'true')) {
```

Comment:
I'm not seeing why we would support this as an option, since then it'd apply to all the resources. If all you need is a proxy, you should use a proxy. The query option is fine, as it's an individual opt-out.